io.gitlab.liferooter.TextPieces.gnome.yaml

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2023 Gleb Smirnov <glebsmirnov0708@gmail.com>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+id: io.gitlab.liferooter.TextPieces.gnome
```

Comment:
Why `.gnome` ? It should just be `io.gitlab.liferooter.TextPieces`

Reply:
Because https://gitlab.com/liferooter/texpieces is a library and this application is its front-end. For now it's the only one, but in the future maybe I will create more for more platforms and on more stacks, not only on GNOME's one. Technically the source code of this app is located at https://gitlab.com/liferooter/textpieces-gnome, but I link the library in the ID because it feels more natural for me.

Reply:
This accents that the app is UI for https://gitlab.com/liferooter/textpieces for GNOME

Reply:
the library is irrelevant, and the "accent" is irrelevant too. That's not what app-id are. `io.gitlab.liferooter.TextPiecesGnome` would be acceptable albeit a poor choice.

Reply:
The library is the implementation of the application, designed as a library. The UI is just a front-end for the application, which is designed as a library. Potentially one of many. I initially wanted to use `io.gitlab.liferooter.TextPieces` as the app ID, but then decided to add `.gnome` suffix to make it possible to write other front-ends.

I know what the application ID is. It's a unique identifier based on reverse-DNS address of the location where the app is hosted and which the developer has control of. It's unique (that's why I use `.gnome` suffix, to make me able to write more front-ends based on the same back-end), it's based of reverse-DNS address of the location where the app is hosted (its implementation is located at https://gitlab.com/liferooter/textpieces, which controls any sublocation of liferooter.gitlab.io/textpieces), and I have control of it. I don't use an ID based on the gitlab.com/liferooter/textpieces-gnome because the application is divided into two repositories and I think that gitlab.com/liferooter/textpieces is the most important part of it. It's not some library, it's back-end of the application implementation moved in a separate crate to make it possible to write many front-ends.

I know that this case is controversial and `io.gitlab.liferooter.textpieces-gnome` is another possible choice and is a better one from some points of view, but I see no reasons why my variant is not possible too and I personally prefer it. In future it will allow me to have something like `io.gitlab.liferooter.TextPieces.gnome`, `io.gitlab.liferooter.TextPieces.cosmic`, `io.gitlab.liferooter.TextPieces.liferooters-new-desktop-with-its-own-graphical-stack` and so on without losing the idea that it's the same application but with different front-ends.

As an author, I prefer this application ID. Do you see any real problems with it, or just think that it's just better to use another one?

Reply:
Okay, I though about it and decided to change the application ID to `io.gitlab.liferooter.TextPieces`. I'll notify when I finish.

Reply:
I've changed the application ID to io.gitlab.liferooter.TextPieces